components/announcement_bar/configuration_bar/configuration_bar.jsx

```diff
@@ -65,41 +63,76 @@ class ConfigurationAnnouncementBar extends React.PureComponent {
         var canCloseBar = false;
 
         switch (warnMetricStatus.id) {
+        case WarnMetricTypes.SYSTEM_WARN_METRIC_NUMBER_OF_POSTS_500K:
         case WarnMetricTypes.SYSTEM_WARN_METRIC_NUMBER_OF_ACTIVE_USERS_500:
             if (warnMetricStatus.acked) {
-                message = (
-                    <React.Fragment>
-                        <img
-                            className='advisor-icon'
-                            src={ackIcon}
-                        />
-                        <FormattedMarkdownMessage
-                            id={t('announcement_bar.error.number_active_users_warn_metric_status_ack.text')}
-                            defaultMessage={'Your trial has started! Go to the [System Console](/admin_console/environment/web_server) to check out the new features.'}
-                        />
-                    </React.Fragment>
-                );
+                if (isE0Edition) {
+                    message = (
+                        <React.Fragment>
+                            <img
+                                className='advisor-icon'
+                                src={ackIcon}
+                            />
+                            <FormattedMarkdownMessage
+                                id={'announcement_bar.error.number_active_users_warn_metric_status_ack.text'}
+                                defaultMessage={'Your trial has started! Go to the [System Console](/admin_console/environment/web_server) to check out the new features.'}
+                            />
+                        </React.Fragment>
+                    );
+                } else {
+                    message = (
+                        <React.Fragment>
+                            <img
+                                className='advisor-icon'
+                                src={ackIcon}
+                            />
+                            <FormattedMessage
+                                id={'announcement_bar.warn_metric_status_ack.text'}
+                                defaultMessage={'Thank you for contacting Mattermost. We will follow up with you soon.'}
```

Comment:
We're not going to mention that they're going to need to install a different instance? What's up with that?

Reply:
@mkraft, I'm not sure I understand the concern, can you elaborate a bit?

Reply:
We are linking to documentation in almost all cases so that they can explore the feature and determine if it would be valuable to them. I would rather lead with the value of the features in question so they can make that determination for themselves.